src/ctia/events/obj_to_event.clj

```diff
@@ -0,0 +1,61 @@
+(ns ctia.events.obj-to-event
+  (:require [clojure.data :refer [diff]]
+            [schema.core :as s]
+            [ctia.lib.time :as t]
+            [ctia.schemas.actor :refer [StoredActor]]
+            [ctia.schemas.campaign :refer [StoredCampaign]]
+
+            [ctia.events.schemas :as vs]))
+
+
+(s/defn to-create-event :- vs/CreateEvent
+  [model :-  {s/Any s/Any}
```

Comment:
Unfortunately I couldn't find a schema to tell the input is itself a Schema.
As all our schema are maps I used it. But I also could remove the signature to the `model` input.

But I like the ability to tell that `to-create-event` will output a `CreateEvent` instead of anything.
So I'm not sure I'll be at ease to remove all schemas.